alkira/resource_alkira_credential_pan_instance.go

```diff
@@ -44,11 +45,13 @@ func resourceAlkiraCredentialPanInstance() *schema.Resource {
 				Description: "PAN password.",
 				Type:        schema.TypeString,
 				Required:    true,
+				Deprecated:  "Not supported anymore",
```

Comment:
This should be moved to service_pan's instance block.

Reply:
Added username and password to service_pan instance block as optional with deprecated message.